rplugin/python/ensime.py

```diff
@@ -323,12 +315,12 @@ def handle_payload(self, payload):
         typehint = payload["typehint"]
         if typehint == "SymbolInfo":
             try:
-                self.message(payload["declPos"]["file"])
+                #self.message(payload["declPos"]["file"])
                 if self.open_definition:
                     self.clean_errors()
-                    self.vim.command(":{} {}".format("split" if self.split else "e", payload["declPos"]["file"]))
+                    self.vim.command("{} {}".format("split" if self.split else "e", payload["declPos"]["file"]))
                     self.set_position(payload["declPos"])
-                    self.vim.command("filetype detect")
+                    self.vim.command("syntax enable")
```

Comment:
We shouldn't have to force syntax enable here. We need to figure out why syntax highlighting becomes disabled.